app/builders/immediate_email_builder.rb

```diff
@@ -42,8 +42,12 @@ def body(content_change, subscriptions)
       <<~BODY
         #{presented_content_change(content_change)}
         ---
+        You’re getting this email because you subscribed to #{subscriptions.first.subscriber_list.title} updates on GOV.UK.
 
         #{presented_unsubscribe_links(subscriptions)}
+
```

Comment:
Do we need this extra empty new line (and same for the other ones)? I don't think it will show up in the email.

Reply:
The spacing would be useful – will the line breaks in the code get ignored?

Reply:
Empty line breaks will be ignored, but you can do something like this to force a line break to appear:

```
blah blah

&nbsp;

blah blah
```